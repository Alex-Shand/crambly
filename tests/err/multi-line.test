# If a command needs to span multiple lines, continuation lines must start with
# >
$ for i in $(seq 1 10); do
>     echo "$i"
> done

# Evens only!
2
4
6
8
10