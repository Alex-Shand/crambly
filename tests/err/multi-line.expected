# If a command needs to span multiple lines, continuation lines must start with
# >
$ for i in $(seq 1 10); do
>     echo "$i"
> done

# Evens only!
1
2
3
4
5
6
7
8
9
10